src/components/Layout/Navbar/components/UserProfile.tsx

```diff
@@ -1,34 +1,40 @@
-import { useGetSelfDetailsQuery } from "../../../../services/serverApi";
 import { Box, Text } from "@chakra-ui/react";
 import Image from "next/image";
+import { useSelector } from "react-redux";
+import { RootState } from "@/src/store";
 
 import { Dispatch, SetStateAction } from "react";
 
 import styles from "./userProfile.module.css";
 
-export default function UserProfile({ isDropdownVisible, setIsDropdownVisible} : {
+export default function UserProfile({
+  isDropdownVisible,
+  setIsDropdownVisible,
+}: {
   isDropdownVisible: boolean;
   setIsDropdownVisible: Dispatch<SetStateAction<boolean>>;
 }) {
-  const { data: user, isLoading } = useGetSelfDetailsQuery();
-  const imageToShow = user?.picture?.url || '/images/Avatar.png';
+  const { first_name, imageURL } = useSelector(
+    (state: RootState) => state.global
+  );
+
+  const imageToShow = imageURL || "/images/Avatar.png";
```

Comment:
```suggestion
  const imageToShow = imageURL || dummyImage;
```

Reply:
Is it possible to import an image into a constant variable and then use it?

Reply:
sure

Reply:
anyway this component is getting deleted in next PR https://github.com/Real-Dev-Squad/members-site/pull/121